src/scripts/main.js

```diff
@@ -22,7 +22,22 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
-}
+  const dataKeys = Object.keys(data);
+  const ul = document.createElement('ul');
+
+  if (dataKeys.length === 0) {
```

Comment:
The check `dataKeys.length === 0` is unnecessary. If `dataKeys` is empty, the for loop on line 13 won't execute, making the check redundant. You can safely remove this check.

Reply:
but without this check it appends unnecessary ul